google/cloud/forseti/common/gcp_api/storage.py

```diff
@@ -347,10 +354,17 @@ def download_to_file(self, bucket, object_name, output_file):
         media_request.http = self.http
 
         downloader = http.MediaIoBaseDownload(output_file, media_request)
-        done = False
-        while not done:
-            progress, done = downloader.next_chunk(
-                num_retries=self._num_retries)
+        try:
+            while not done:
+                progress, done = downloader.next_chunk(
+                    num_retries=self._num_retries)
+        except errors.HttpError as e:
+            if e.resp.status in ignored_http_error_codes:
```

Comment:
This is the fix for downloading empty files from GCS storage. If the org/access policy files are empty, then an exception is raised and the inventory process stops. [Others have hit this issue with GCS](https://github.com/googleapis/google-api-python-client/issues/638#issuecomment-574407928), no apparent fix atm.